tests/main/apparmor-prompting-integration-tests/task.yaml

```diff
@@ -0,0 +1,110 @@
+summary: Check that AppArmor Prompting works end-to-end
+
+details: |
```

Comment:
this should have a a link to something that explains the scripting language of the prompting client?

Reply:
Sure, I'll ask Innes if there's documentation of the json format. Is that what you mean by scripting language? The only other script-related thing would be the `--var` argument, which performs bash-style variable substitution where `--var "FOO:BAR"` sets every instance of `$FOO` or `${FOO}` to `BAR` in the the script. Strange that there's no help text for that arg:

```
me@hostname:~$ prompting-client.scripted --help
Run a scripted client expecting a given sequence of prompts

Usage: prompting-client-scripted [OPTIONS] --script <FILE>

Options:
  -v, --verbose...              
  -s, --script <FILE>           The path to the input JSON file
      --var <VAR>               
  -g, --grace-period <SECONDS>  The number of seconds to wait following completion of the script to check for any unexpected additional prompts
  -h, --help                    Print help
```

Reply:
https://github.com/canonical/prompting-client/blob/main/docs/running-the-scripted-client.md